.yarn/patches/@docusaurus-theme-common-npm-2.1.0-d5ae2a9539.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/lib/utils/docsUtils.js b/lib/utils/docsUtils.js
+index 61d72a3300b94904c8809503360206081e1c029c..43d75f84e8fc18e023ebb91afc17bd7354d9be27 100644
+--- a/lib/utils/docsUtils.js
++++ b/lib/utils/docsUtils.js
+@@ -105,13 +105,13 @@ export function isActiveSidebarItem(item, activePath) {
+  * Get the sidebar the breadcrumbs for a given pathname
+  * Ordered from top to bottom
+  */
+-function getSidebarBreadcrumbs({ sidebarItems, pathname, onlyCategories = false, }) {
++function getSidebarBreadcrumbs({ sidebarItems, pathname, hash = '', onlyCategories = false, }) {
```

Comment:
is there a plan to trigger an issue to docusaurus about this ? it doesn't seem sustainable to keep this as a patch

Reply:
or even an issue to track ejecting the breadcrumbs completely

Reply:
@ling1726 I created an issue to track it #219.